data/sql/updates/pending_db_world/rev_1609174873784850500.sql

```diff
@@ -0,0 +1,29 @@
+-- INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609174873784850500');
+
+UPDATE `creature_template_addon` SET `auras` = '29266' WHERE (`entry` = 17508);
```

Comment:
Are the parenthesis needed from the where clause or is this the code convention for AC?
Same question on line: 8, 29.

Reply:
Not needed, some just use parenthesis

Reply:
This is just the default of Keira